docs/usage-guide/topics/ch06-security-policies.md

```diff
@@ -42,7 +43,10 @@ The following chart maps the security policy version to protocol version and cip
 The "default", "default_tls13", and "default_fips" versions are special in that they will be updated with future s2n-tls changes and ciphersuites and protocol versions may be added and removed, or their internal order of preference might change. Numbered versions are fixed and will never change.
 In general, customers prefer to use numbered versions for production use cases to prevent impact from library updates.
 
-"20230317" is a FIPS compliant policy. It offers more limited but more secure options than "default". It only supports TLS1.2 and TLS1.3. Consider this policy if you plan to enable FIPS mode or don't need or want to support less secure legacy options like TLS1.1 or SHA1.
+"20230317" offers more limited but more secure options than the default policies.
+Consider it if you don't need or want to support less secure legacy options like
```

Comment:
nit: For markdown files I prefer not to use line breaks and just let editors reflow text as necessary. That way the person who edits this doesn't have to tamper with all of the line breaks again.

Reply:
Force of habit lol. Removed the line breaks.